assets/scss/components/_reminders.scss

```diff
@@ -0,0 +1,131 @@
+/*
+Reminder List
+
+Display an Unordered List of reminder items with time formatted in a new line.
+
+Markup:
+<ul class="reminder-list">
+  <li>Buy Envelope</li>
+  <li>Buy Stamp</li>
+  <li>Post form by <time>5:00PM</time></li>
+</ul>
+
+
+Styleguide Reminders.reminder-list
+*/
+
+
+.reminder-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  li {
+    padding: 0.5em 0;
+
+    time {
+      display: block;
+      color: $secondary-text-colour;
+
+      @include core-16;
+    }
+  }
+}
+
+
+/*
+Reminder
+
+Formats containing elements in a blue background and adds spacing.
+
+Markup:
+<div>
+  <div class="reminder">
+    <div class="label">Reminder label</div>
+    <p>This is text in a reminder paragraph.</p>
+     <a href="#">Links are shown like this in a new line</a>
+     <div class="action">Adding action class adds an underline style</div>
+  </div>
+  <br>
+ <div class="reminder urgent">
+    <div class="label">Reminder label</div>
+    <p>If you are using a lot of text in the paragraph, note that when it wraps
+    it looks like this and the whole paragraph moved down below label.</p>
+     <a href="#">When urgent is used links are like this</a>
+  </div>
+</div>
+
+.label Can be used in a child element to add a label block
+
+.action Can be used in a child element to provide emphasis
```

Comment:
`.action` and `.label` can either be described in the description or just as elements